Shaders/Materials/BlinnPhong/BlinnPhongMaterial.glsl

```diff
@@ -98,16 +98,18 @@ vec3 getSpecularColor(Material material, vec2 texC) {
 
 vec3 computeMaterialInternal(Material material, vec2 texC, vec3 L, vec3 V, vec3 N, vec3 X, vec3 Y) {
     vec3 H =  normalize(L + V);
-    // http://www.thetenthplanet.de/archives/255 / VortexEngine BlinnPhong
+    // http://www.thetenthplanet.de/archives/255
     float Ns = getNs(material, texC);
     vec3 Kd = getKd(material, texC) / Pi;
-    float normalization = ((Ns + 2) * (Ns + 4)) / (8 * Pi * (exp2(-Ns * 0.5) + Ns));
+
+    // use the correct normalization factor for Blinn-Phong BRDF;
+    float normalization = (Ns + 1) / (8 * Pi * pow(dot(L, H), 3));
```

Comment:
Not sure where this one come from (and I do not take too much time on trying to understand).
It might be usefull to add a ref here, and for instance why it's not what I understand form [this explanation](https://seblagarde.wordpress.com/2011/08/17/hello-world/)

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.

Reply:
The ref is given in the shader : it comes from "The Blinn-Phong Normalization Zoo", http://www.thetenthplanet.de/archives/255 , mainly from the "minimalist cook-torrance" view of Blinn-Phong.
The explanations you give is also debated in "The Blinn-Phong Normalization Zoo". This normalization allow to integrate a better "Fresnel Like" effect at grazing angles.